Jenkinsfile

```diff
@@ -83,6 +86,10 @@ def cmd_sanitize(String cmd){
   return true
 }
 
+def gitNotify (context, description, status, targetUrl='' ){
```

Comment:
What do you about implementing `Notification` as an abstract class? In the future we would need email, IM notifications (maybe some others). Each notification provider would've done its own class implementation. This would allow for a more uniform behaviour across different notifiers.